rl_loop/distributed_flags

```diff
@@ -8,6 +8,8 @@
 --conv_width=256
 --fc_width=256
 --trunk_layers=19
+--use_SE
```

Comment:
Should we set these default true so that the python code runs while v17 is active

Reply:
probably not; most folks will use our v15 models until this has come along farther.